src/LightstonePlatform.Products/Models/ProductFlowHistoryModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using LightstonePlatform.Products.Enums;
+using Newtonsoft.Json;
+using System;
+
+namespace LightstonePlatform.Products.Models
+{
+    public class ProductFlowHistoryModel
```

Comment:
Should the `Model` not be dropped to align to the other changes in this PR?

It is also already in a `Models` namespace

Reply:
Have removed the Model postfix as per the comment